docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -362,6 +362,16 @@ TestCafe actions can interact with elements if they satisfy the following condit
 
 ## Examples
 
+### Disable the 'Close Multiple Tabs' Popup in IE
```

Comment:
Seems like it doesn't go here after all. First, this is not a TestCafe example. Second, it doesn't relate much to any TestCafe action in particular or actions in general.

(the second example about downloading serves to emulate a user action, while our example just works around a limitation to run tests)

So, I guess it should be a `## Troubleshooting` section somewhere in the `Run Tests` guide.

Reply:
moved